### PR TITLE
Adding bootstrap to proyect

### PR DESCRIPTION
**Pull request by jshidalgokommit**
bootstrap implemented in the blog project

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 # articles comments controller
 class CommentsController < ApplicationController
-  http_basic_authenticate_with name: 'admin', password: 'admin', only: :destroy
+  # http_basic_authenticate_with name: 'admin', password: 'admin', only: :destroy
```
Comment (JoseCastroA) [inline app/controllers/comments_controller.rb:5]: @jshidalgokommit Do we need this comment?

File: app/views/articles/edit.html.erb [hunk 2]
```diff
@@ -1,3 +1,5 @@
-<h1>New Article</h1>
+<%= render 'layouts/breadcrumb', data: {:current => 'Edit article', :breadcrumb => [{:name => 'Home', :url => root_path}]} %>
+
+<h1 align="center">Edit Article</h1>
 
 <%= render "form", article: @article %>
```
Comment (JoseCastroA) [inline app/views/articles/edit.html.erb:5]: @jshidalgokommit Please add an empty line

File: app/views/articles/index.html.erb [hunk 3]
```diff
@@ -1,15 +1,19 @@
...
+<h1 align="center">Articles</h1>
 
-<ul>
+<div class="list-group">
   <% @articles.each do |article| %>
     <% unless article.archived? %>
-      <li>
-        <%= link_to article.title, article %>
-      </li>
+      <a href="<%= article_path article %>" class="list-group-item list-group-item-action" aria-current="true">
```
Comment (JoseCastroA) [inline app/views/articles/index.html.erb:8]: @jshidalgokommit Please use link_to instead of href

File: app/views/articles/new.html.erb [hunk 4]
```diff
@@ -1,3 +1,5 @@
-<h1>New Article</h1>
+<%= render 'layouts/breadcrumb', data: {:current => 'New article', :breadcrumb => [{:name => 'Home', :url => root_path}]} %>
+
+<h1 align="center">New Article</h1>
 
 <%= render "form", article: @article %>
```
Comment (JoseCastroA) [inline app/views/articles/new.html.erb:5]: @jshidalgokommit Please add an empty line

File: app/views/articles/show.html.erb [hunk 5]
```diff
@@ -1,17 +1,38 @@
...
+
+    </div>
+
+  </div>
+</div>
+
+<div class="m-2 p-2">
+  <h5>Comments</h5>
+
+  <% if !@article.comments.empty? %>
```
Comment (JoseCastroA) [inline app/views/articles/show.html.erb:28]: @article.comments.present?

File: app/views/layouts/application.html.erb [hunk 6]
```diff
@@ -1,16 +1,43 @@
...
+    <a class="navbar-brand" href="<%= root_path %>">Articles Blog</a>
+    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarSupportedContent">
+      <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+
+      </ul>
+      <div>
+        <a type="button" class="btn btn-success" href="<%= new_article_path %>">New post</a>
```
Comment (JoseCastroA) [inline app/views/layouts/application.html.erb:30]: link_to

**Review by JoseCastroA — CHANGES_REQUESTED** [review]

**Comment by jshidalgokommit** [comment]
Hello @JoseCastroA, changes requests have been implemented

**Review by JoseCastroA — APPROVED** [review]

**Pull request merged**